app/commands/v2/discard_draft.rb

```diff
@@ -47,7 +47,7 @@ def delete_draft_from_draft_content_store(draft_path)
       def send_live_to_draft_content_store(live)
         PresentedContentStoreWorker.perform_async(
           content_store: Adapters::DraftContentStore,
-          payload: Presenters::ContentStorePresenter.present(live, event),
+          payload: Presenters::ContentStorePresenter.present(live, event, [:draft, :published]),
```

Comment:
OK, that makes sense.  Could we make it a named parameter so it's clearer at call time?